dashboard/src/components/ApplicationStatus/ApplicationStatus.tsx

```diff
@@ -134,10 +134,15 @@ export default function ApplicationStatus({
       </div>
     );
   }
-  if (info && info.status) {
+  if (info?.status?.reason) {
     // If the status code is different than "Deployed", display that status
```

Comment:
Shouldn't we update this comment to include the pending status added, or it is not related to the `if` condition modified?

Reply:
Woops, yep, I'll update it. Thanks.